packages/runtime/container-runtime/api-report/container-runtime.api.md

```diff
@@ -78,6 +78,9 @@ export enum CompressionAlgorithms {
     lz4 = "lz4"
 }
 
+// @alpha (undocumented)
```

Comment:
Please be sure to add appropriate documentation to all of the new `undocumented` API members here.